### PR TITLE
Webuipoc: Initial PR

### PR DESCRIPTION
**Pull request by psiinon**
## Overview
First phase of an add-on that GSoC contributors can use to create PoCs for the Web based UI project.

## Checklist
- [ ] Update help
- [ ] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [ ] Write tests
- [ ] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

For more details, please refer to the [developer rules and guidelines](https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/).

### CODE REVIEW

**Comment by psiinon** [comment]
Ugh, forgot the change the help .. will do that ..

**Comment by psiinon** [comment]
Help removed, looking better now 😉

File: addOns/webuipoc/src/main/java/org/zaproxy/addon/webuipoc/ExtensionWebUiPoc.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2014 The ZAP Development Team
```
Comment (kingthorin) [inline addOns/webuipoc/src/main/java/org/zaproxy/addon/webuipoc/ExtensionWebUiPoc.java:6]: 2024

Comment (kingthorin) [reply addOns/webuipoc/src/main/java/org/zaproxy/addon/webuipoc/ExtensionWebUiPoc.java:6]: All classes it seems

Comment (psiinon) [reply addOns/webuipoc/src/main/java/org/zaproxy/addon/webuipoc/ExtensionWebUiPoc.java]: Fixed 😁

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]
Thanks!!!!

**Pull request merged**